app/models/user.rb

```diff
@@ -1,12 +1,25 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
+  before_validation :default_values
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 I have successfully configured your device 👍🏻 But an error message is displayed when the confirmation link has been sent and when the reset password link is sent, ensure you prevent this 👍🏻


![18 1](https://user-images.githubusercontent.com/79058364/192903345-120be867-54d9-4f05-ad27-dd423d3251ce.png)

Reply:
Thanks, @BenMukebo, I'll work on fixing this.